conf/configuration.go

```diff
@@ -528,6 +529,8 @@ func init() {
 	viper.SetDefault("inspect.backloglimit", consts.RequestThrottleBacklogLimit)
 	viper.SetDefault("inspect.backlogtimeout", consts.RequestThrottleBacklogTimeout)
 
+	viper.SetDefault("lyricspriority", "embedded,.lrc,.txt")
```

Comment:
I think the default should be `.lrc,.txt,embedded`, to maintain consistency with coverart (`cover.*, folder.*, front.*, embedded, external`)